lib/rules/no-unused-placeholders.js

```diff
@@ -0,0 +1,107 @@
+/**
+ * @fileoverview Disallow unused placeholders in rule report messages
+ * @author 薛定谔的猫<hh_2013@foxmail.com>
+ */
+
+'use strict';
+
+const utils = require('../utils');
+
+// ------------------------------------------------------------------------------
+// Rule Definition
+// ------------------------------------------------------------------------------
+
+/**
+* Gets report info for a context.report() call
+* TODO: Combine this with logic from no-deprecated-report-api
+* @param {ASTNode[]} reportArgs The arguments to the context.report() call
+* @returns {object}
+*/
+function getReportInfo (reportArgs) {
+  if (reportArgs.length === 1) {
```

Comment:
Is it possible to use [`utils.getReportInfo`](https://github.com/not-an-aardvark/eslint-plugin-eslint-plugin/blob/8ac484c68e7314d6977855c1901541f6a3f5dc20/lib/utils.js#L192) for this?